sleap/io/cameras.py

```diff
@@ -883,13 +943,13 @@ def update_points(
             if not isinstance(instance, PredictedInstance):
                 instance_oks = compute_oks(
                     gt_points[cam_idx, :, :],
-                    points[cam_idx, :, :],
+                    points_3d[cam_idx, :, :],
```

Comment:
Here we want to use `points_reprojected`
```suggestion
                    points_reprojected[cam_idx, :, :],
```